### PR TITLE
Don't dump object internals

### PR DESCRIPTION
**Pull request by jonasdt**
See discussion in #401

### CODE REVIEW

File: src/Whoops/Handler/PrettyPageHandler.php [hunk 1]
```diff
@@ -532,4 +538,14 @@ public function setResourcesPath($resourcesPath)
...
         $this->addResourcePath($resourcesPath);
     }
+
+    /**
+     * Caster that removes all internals.
+     *
+     * @see AbstactCloner
+     * @return array
+     */
+    public function castObject($obj, $a, $stub, $isNested) {
```
Comment (staabm) [inline src/Whoops/Handler/PrettyPageHandler.php]: Could we use a closure instead so we dont need this additional public api method?

**Comment by staabm** [comment]
Instead of only showingnthe most top lvl could we get a bit more detailed and inspect at least 1 level deeper?

**Comment by jonasdt** [comment]
That was also my first idea last week, but it's not so easy ...

File: src/Whoops/Util/TemplateHelper.php [hunk 2]
```diff
@@ -110,10 +116,9 @@ public function dump($value)
         $dumper = $this->getDumper();
 
         if ($dumper) {
-            $cloner = new VarCloner();
 
             // re-use the same DumpOutput instance, so it won't re-render the global styles/scripts on each dump.
-            $dumper->dump($cloner->cloneVar($value), $this->htmlDumperOutput);
+            $dumper->dump($this->cloner->cloneVar($value), $this->htmlDumperOutput);
```
Comment (denis-sokolov) [inline src/Whoops/Util/TemplateHelper.php]: Perhaps this is supposed to be `this->getCloner()`?

Comment (jonasdt) [inline src/Whoops/Util/TemplateHelper.php]: Correct, I've fixed it.

**Comment by denis-sokolov** [comment]
Did you close it by accident?

**Comment by jonasdt** [comment]
Nope, I'll make a new pull request.

**Pull request closed without merging**